_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -25409,8 +25409,8 @@
 /area/station/science/xenobiology)
 "gmK" = (
 /obj/structure/cable,
-/obj/machinery/newscaster/directional/south,
 /obj/effect/turf_decal/tile/yellow/half/contrasted,
+/obj/machinery/mining_weather_monitor/directional/south,
```

Comment:
could this be the reason mapdiff is choking
maybe try reverting the map changes and commit them again 🤷